package.json

```diff
@@ -24,10 +24,13 @@
     "winston": "^3.8.2"
   },
```

Comment:
add `"postinstall": "husky install"` under scripts objects to solve the husky not installed when installing the dependency

Reply:
fixed the issue of dependency check it now